src/Goteo/Model/Faq/FaqSubsection.php

```diff
@@ -96,54 +96,68 @@ static public function getList(array $filters = [], int $offset = 0, int $limit
         return $query->fetchAll(\PDO::FETCH_CLASS, __CLASS__);
     }
 
-  public function getFaqbySubsection($limit=0){
+    public function getListCount(array $filters = [], $lang = null): int {
+        if(!$lang) $lang = Lang::current();
+        list($fields, $joins) = self::getLangsSQLJoins($lang, Config::get('sql_lang'));
 
-    return Faq::getList(['subsection'=>$this->id], 0, $limit);
+        $filter = [];
+        $values = [];
 
-  }
+        if ($filters['section']) {
+            $filter[] = "faq_subsection.section_id = :section_id";
+            $values[':section_id'] = $filters['section'];
+        }
 
-  /**
-   * Save.
-   *
-   * @param   type array  $errors
-   * @return  type bool   true|false
-   */
-  public function save(&$errors = array()) {
+        if($filter) {
+            $sql = " WHERE " . implode(' AND ', $filter);
+        }
 
-    if (!$this->validate($errors))
-        return false;
+        $sql = "SELECT
+                      count(faq_subsection.id)
+                  FROM faq_subsection
+                  $joins
+                  $sql";
 
-    $fields = [
-        'id',
-        'section_id',
-        'name',
-        'lang',
-        'order'
-        ];
+        $query = static::query($sql, $values);
+        return $query->fetchColumn();
+    }
+
+    public function getFaqs(): Faq
+    {
+        $total = Faq::getListCount(['subsection' => $this->id]);
+        return Faq::getList(['subsection'=>$this->id], 0, $total);
+    }
 
-    try {
-        //automatic $this->id assignation
-        $this->dbInsertUpdate($fields);
+    public function save(&$errors = array()): bool
+    {
 
-        return true;
-    } catch(\PDOException $e) {
-        $errors[] = "Faq subsection save error: " . $e->getMessage();
-        return false;
+        if (!$this->validate($errors))
+            return false;
+
+        $fields = [
+            'id',
+            'section_id',
+            'name',
+            'lang',
+            'order'
+            ];
```

Comment:
@davidbeig maybe you would like to move this variable as a class constant too. As I proposed you in `FaqSection` class.